File: android/src/main/java/ru/vvdev/yamap/models/RNMarker.java
package ru.vvdev.yamap.models;

public class RNMarker {
    public double lon;
    public double lat;
    public String id;
    public Integer zIndex;
    public String uri;
    public RNMarker(double _lon, double _lat, String _id, Integer _zIndex, String _uri) {
        lon = _lon;
        lat = _lat;
        id = _id;
        zIndex = _zIndex;
        uri = _uri;
    }
}

File: ios/Models/RNMarker.m
#import "RNMarker.h"


@implementation RNMarker {
}
- (RNMarker*)initWithJson:(NSDictionary *)json {
    self.lon = [[json valueForKey:@"lon"] doubleValue];
    self.lat = [[json valueForKey:@"lat"] doubleValue];
    self._id = [json valueForKey:@"id"];
    self.uri = [json valueForKey:@"source"];
    self.zIndex = [[json valueForKey:@"zIndex"] intValue];
    return self;
}

@end
